spec/spec_helper.rb

```diff
@@ -13,3 +15,9 @@
     c.syntax = :expect
   end
 end
+
+def to_xml(parsed_schema, escape_content_tags: false)
```

Comment:
@suleman-uzair can you see if the [`xml-c14n` gem](https://github.com/metanorma/xml-c14n) helps? We want to share code across all of these gems. Thanks.

Reply:
> can you see if the xml-c14n[ gem](https://github.com/metanorma/xml-c14n) helps?

@ronaldtse I have replaced `equivalent-xml` with `xml-c14n` gem, it works fine but the `gsub!` used in this method is still needed though.
The reason for using `gsub` is that when using `map_all` to process `documentation` tags in the schema that contain elements (tags), it is treated as text, which converts `<` and `>` to `&lt;` and `&gt;`, respectively, causing a failure in the matcher.